### DISCUSSION TITLE
Problem with install ps3netsrv

### DISCUSSION THREAD

**Discussion by rumun1990** [retronas/discussions#45] [General]
///EDIT

After hard reset , installation was done :)

Hi,
i'd like to install ps3netsrv, but i have these errors. Any ideas how to resolve this ?

 TASK [Generic ROM dir - build layout] *******************************************************************************************************************************************************
task path: /opt/retronas/ansible/retronas_create_dirs.yml:7
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'retronas_group' is undefined\n\nThe error appears to be in '/opt/retronas/ansible/retronas_create_dirs.yml': line 7, column 3, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n- name: \"{{ my_name }} - build layout\"\n  ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

PLAY RECAP **********************************************************************************************************************************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

and another one

TASK [ps3netsrv - build top level] **********************************************************************************************************************************************************
task path: /opt/retronas/ansible/install_ps3netsrv.yml:43
fatal: [localhost]: FAILED! => {"msg": "The task includes an option with an undefined variable. The error was: 'retronas_group' is undefined\n\nThe error appears to be in '/opt/retronas/ansible/install_ps3netsrv.yml': line 43, column 5, but may\nbe elsewhere in the file depending on the exact syntax problem.\n\nThe offending line appears to be:\n\n\n  - name: \"{{ my_name }} - build top level\"\n    ^ here\nWe could be wrong, but this one looks like it might be an issue with\nmissing quotes. Always quote template expression brackets when they\nstart a value. For instance:\n\n    with_items:\n      - {{ foo }}\n\nShould be written as:\n\n    with_items:\n      - \"{{ foo }}\"\n"}

PLAY RECAP **********************************************************************************************************************************************************************************
localhost                  : ok=4    changed=0    unreachable=0    failed=1    skipped=0    rescued=0    ignored=0

Press [Enter] to continue...

**Comment by sairuk** [comment]
you've run into a known bug with the last retronas release where the `retronas_vars.yml` could end up with multiple `retronas_group` entries, one being blank, we implemented a fix for that once it was found which is why a hard reset worked for you.

alternatively you could've removed the blank `retronas_group` entry from the `/opt/retronas/ansible/retronas_vars.yml` with any text editor and the install would have then worked
